specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineImage.json

```diff
@@ -665,6 +665,59 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/edgeZones/{edgeZone}/vmimages": {
```

Comment:
>subscriptionId [](http://example.com/codeflow?start=21&length=14)

Adding an operation to an existing api version constitutes a breaking change. You will have to introduce this in a new API version